src/logic/LocalMessageCache.tsx

```diff
@@ -132,6 +137,23 @@ class LocalMessageCache {
         return;
     }
 
+    static all(
```

Comment:
I guess if we merge #151 first, then we can rebase this PR and this will be removed from the diff?

Reply:
Hope so!